2019/python/rules_engine.py

```diff
@@ -1125,10 +1125,10 @@ def v648_1(self):
 	def v648_2(self):
 		"""An invalid Sex data field was reported.
 		2) If Sex of Co-Applicant or Co-Borrower equals 6, then
-		Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname must equal 2."""
+		Sex of Co-Applicant or Co-Borrower Collected on the Basis of Visual Observation or Surname must equal 2 or 3."""
 		field = "Co-Applicant Sex Basis"
```

Comment:
Please add a comment that the inclusion of 3 is from: https://github.com/cfpb/hmda-platform/issues/2774

Reply:
change has been implemented in recent commit.